packages/react-devtools-shared/src/backend/DevToolsFiberComponentStack.js

```diff
@@ -53,7 +52,6 @@ export function describeFiber(
     case SuspenseListComponent:
       return describeBuiltInComponentFrame('SuspenseList', owner);
     case FunctionComponent:
-    case IndeterminateComponent:
```

Comment:
We should keep it here for compatibility with React versions < 19

Reply:
good call, thanks!